### ISSUE TITLE
error when feGaussianBlur in svg

### ISSUE THREAD

**Issue by p-art** [canvg/canvg#580]
When simple feGaussianBlur is used in svg, canvg calculates/extracts huge numbers for height, width and blurRadius. Those values are then passed to stackBlur.canvasRGBA() which throws "Arithmetic result exceeded 32 bits." Here is svg that leads to the error:

``
<svg id="svg0" xmlns="http://www.w3.org/2000/svg" version="1.1" width="150" height="150">
  <defs>
        <filter id="f0">
          <feGaussianBlur in="SourceGraphic" stdDeviation="1.80"></feGaussianBlur>
        </filter> 
  </defs>
  <rect x="100" y="100" width="40" height="20" rx="3" ry="3" fill="#34495e" filter="url(#f0)"></rect>       
</svg>   
``

**Comment by p-art** [comment]
I've made very important discovery that is worth to share before I remove this error entry. To make it clear, my original claim this is an issue in canvg was incorrect. I apologize about that. Here is what the problem is really about:

Huge decimal value of stdDeviation that breaks stackBlur comes as a result of using cloneNode() in Internet Explorer.  I am using saveSvgAsPng library which passes svg to canvg by replicating source with  cloneNode(). Apparently, cloneNode() in IE changes values of some attributes, such as "stdDeviation" in feGaussianBlur or "radius" in feMorphology to very large decimals. It looks like regardless of original value, cloneNode() in IE converts values of feGaussianBlur .stdDeviation and feMorphology.rdious to 1.88472e+009" in the clone. This sounds ridiculous but I have found someone on stackoverflow reporting the same problem: https://tinyurl.com/y9r8evll
